src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="/"
```

Comment:
Please, check your code style 
![Screenshot 2023-03-03 at 17 52 56](https://user-images.githubusercontent.com/6526149/222765763-67709784-5cab-410a-9352-57d440137173.png)

Reply:
Thanks for your quck review. I've fixed code style in html. But I have one question - why did test in github not complete? Test shows, that is different in pixels? I couln't see, what difference between my work and reference? Thanks!